### ISSUE TITLE
Save/Load Books

### ISSUE THREAD

**Issue by TheRedHippo** [THEREALWWEFAN231/Inertia#972]
**Minecraft Version(s)** All

**What is the suggestion** When opening a saved book, a button will appear in the corner allowing you to save said book. When opening a book and quill, there will be a list of the titles of saved books. Clicking them will automatically load the book.

**Why would this feature be useful**
Being able to get books on any server. Nobody wants to write the entire bee movie script on minecraft books TWICE.

**Comment by feliscatusmeows** [comment]
Similar to forgehax bookbot, which reads from a file?

**Comment by TheRedHippo** [comment]
> Similar to forgehax bookbot, which reads from a file?

Yes, exactly. This suggestion could be implemented in .commands and not necessarily a gui. That would be easier now that I think about it.
The .randombook command would be renamed to just .book and have 2 new commands; 
.book load {bookname}
.book save {bookname}
